### PR TITLE
Footer logo

### PR DESCRIPTION
**Pull request by JGeiser9**
Quick update to replace old footer logos and styling

### CODE REVIEW

File: src/components/layout/Footer.js [hunk 1]
```diff
@@ -144,7 +152,9 @@ export default function Footer() {
     <FooterContainer>
       <ContentContainer>
         <LeftContent>
-          <Logo />
+          <LogoContainer>
+            <LogoText />
+          </LogoContainer>
```
Comment (kilpatty) [inline src/components/layout/Footer.js]: Can we just drop LogoContainer and wrap LogoText with the margin-bottom and width instead?

File: src/components/layout/Footer.js [hunk 2]
```diff
@@ -40,12 +45,15 @@ export const ContentContainer = styled.div`
...
+  flex-direction: column;
   text-align: center;
+  align-items: center;
   max-width: 200px;
 
   @media (min-width: 1024px) {
     text-align: left;
-    flex-direction: row;
+    align-items: flex-start;
     min-width: 200px;
```
Comment (kilpatty) [inline src/components/layout/Footer.js]: Do these change anything (this plus the above)?

File: src/components/svg/LogoText.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const LogoText = () => (
+  <svg width="100%" height="100%" viewBox="0 0 581 82" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink">
```
Comment (kilpatty) [inline src/components/svg/LogoText.js]: This needs Props to be passed down like the deleted HNScan.js component

Comment (JGeiser9) [reply src/components/svg/LogoText.js]: updated with props and theming

File: src/components/layout/Footer.js (hunk 2 above)
Comment (JGeiser9) [reply src/components/layout/Footer.js]: removed from code

File: src/components/layout/Footer.js (hunk 1 above)
Comment (JGeiser9) [reply src/components/layout/Footer.js]: reverted to previous version without container

**Pull request merged**